src/core/a-assets.js

```diff
@@ -163,18 +164,30 @@ function mediaElementLoaded (el) {
         resolve();
       }
     }
+
+    // Respect WebKit's user-gesture requirement for auto-playback of
+    // video containing audio. For more info:
+    // https://webkit.org/blog/6784/new-video-policies-for-ios/
+    window.addEventListener('click', handleFirstClick);
+    function handleFirstClick () {
+      try {
+        if (el.paused) { el.play(); }
```

Comment:
it's already being checked [above](https://github.com/aframevr/aframe/pull/2657/files#diff-565a65bb6612d5571efb7439e5423720R139)